pkg/multicluster/access/errors.go

```diff
@@ -0,0 +1,35 @@
+package access
```

Comment:
what do you think of the name `kubeconfig` for this package? considering that we are exporting functions like `KubeConfigToSecret`, `access` seems a bit vague. wdyt?

Reply:
i was trying to be clever / avoid future references like "kubeconfig.KubeConfigToSecret", and maybe account for non-kubernetes client configs, but we can cross that bridge when we get to it. will rename for clarity.